res/css/structures/auth/_SetupEncryptionBody.scss

```diff
@@ -18,4 +18,5 @@ limitations under the License.
     @mixin ButtonResetWarning;
     color: $light-fg-color; // .mx_Dialog on _common.scss
     margin-top: $font-14px;
+    line-height: 1; // align the warning icon
```

Comment:
This change should fix this bug:

![after (copy 1)](https://user-images.githubusercontent.com/3362943/161339666-e06ff718-e5ae-41e8-9603-06899218a5e3.png)